packages/astro/src/@types/astro.ts

```diff
@@ -438,6 +438,23 @@ export interface AstroUserConfig {
 	 */
 	site?: string;
 
+
+	/**
+	 * @docs
+	 * @name compressHTML
+	 * @type {boolean}
+	 * @default `true`
+	 * @description
+	 * Set to `true` to enable compressHTML mode. This will remove all whitespace from your final HTMl file, including newlines. This is useful for reducing the size of your final build html bundle
+	 *
+	 * ```js
+	 * {
+	 *   compressHTML: true
```

Comment:
Be sure to only show the non-default option here, the one that actually requires being set.

(See my comment above: if the default is true, then there's no reason to show true here as the user would never do that. If the default is actually false, then this example is correct, but the default above needs to be changed.)